bin/prompt_prototype_upload_raws.sh

```diff
@@ -20,37 +20,47 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-# This script uploads the raw files from the HSC PDR2 run to Google Storage. It
-# renames the files to match prompt_prototype conventions. The user must have
-# gsutil already configured.
+# This script uploads the raw files from the HSC PDR2 run to a bucket at USDF.
+# It renames the files to match prompt_prototype conventions. The user must
+# have bucket access already configured.
 
 set -e  # Abort on any error
 
-RAW_DIR="/datasets/hsc/raw/ssp_pdr2/2016-03-07"
-UPLOAD_BUCKET=rubin-prompt-proto-unobserved
+RAW_DIR="/sdf/group/rubin/datasets/hsc/raw/ssp_pdr2/2016-03-07"
+UPLOAD_BUCKET=rubin-pp-users/unobserved
 
 # Filename format is defined in activator/raw.py:
 # instrument/detector/group/snap/exposureId/filter/instrument-group-snap-exposureId-filter-detector
-gsutil cp "${RAW_DIR}/HSCA05913553.fits" \
-    gs://${UPLOAD_BUCKET}/HSC/0/2016030700001/0/0059134/HSC-G/HSC-2016030700001-0-0059134-HSC-G-0.fits
-gsutil cp "${RAW_DIR}/HSCA05913542.fits" \
-    gs://${UPLOAD_BUCKET}/HSC/4/2016030700001/0/0059134/HSC-G/HSC-2016030700001-0-0059134-HSC-G-4.fits
-gsutil cp "${RAW_DIR}/HSCA05913543.fits" \
-    gs://${UPLOAD_BUCKET}/HSC/5/2016030700001/0/0059134/HSC-G/HSC-2016030700001-0-0059134-HSC-G-5.fits
+aws --endpoint-url https://s3dfrgw.slac.stanford.edu s3 \
```

Comment:
Perhaps:
```suggestion
aws --endpoint-url ${S3_ENDPOINT_URL} s3 \
```
combined with a local definition of the variable (per my comment about `aws`)?